files/mcollective/pluginpackager/templates/aiomodule/README.md.erb

```diff
@@ -10,7 +10,8 @@
 
 <%= @plugin.metadata[:description] %>
 
-The <%= module_name %> module is based on the source from <%= @plugin.metadata[:url] %>.
+The <%= module_name %> module generated automatically, based on the source from <%= @plugin.metadata[:url] %>.
+You can find templates used for generation in https://github.com/choria-io/puppet-mcollective/tree/master/files/mcollective/pluginpackager/templates/aiomodule.
```

Comment:
hmm, I really wouldn't want this source link on every single generated package.  These files are likely to move around etc.

I get that it was annoying to find this file, but the generated modules are end user consumables so the messaging there should be appropriate to the non choria dev audience